### DISCUSSION TITLE
Github Stats refresh

### DISCUSSION THREAD

**Discussion by SillySoon** [VishwaGauravIn/github-profile-readme-maker#14] [General]
Hello! Just wondering...

Does the Github Stats feature refreshes it self?
And if yes after how much time? o.o

**Comment by VishwaGauravIn** [comment]
> Hello! Just wondering...
> 
> Does the Github Stats feature refreshes it self? And if yes after how much time? o.o

Yes, Default is 4 hours but you can set it according to your choice.
